File: push/src/main/java/top/trumeet/mipushframework/help/HelpActivity.java
package top.trumeet.mipushframework.help;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class HelpActivity extends AppCompatActivity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        HelpPageKt.init(this);
    }
}

File: legacy/src/main/java/com/xiaomi/mipush/sdk/help/HelpActivity.kt
package com.xiaomi.mipush.sdk.help
import io.github.magisk317.mipush.protocol.model.*

import android.app.Activity
import android.os.Bundle
import com.xiaomi.mipush.sdk.AwakeHelper

class HelpActivity : Activity() {
    override fun onCreate(bundle: Bundle?) {
        super.onCreate(bundle)
        AwakeHelper.doAWork(this, intent, null)
        finish()
    }
}
